src/modules/endpoint/endpoints/sprungbrett.js

```diff
@@ -0,0 +1,17 @@
+import EndpointBuilder from '../EndpointBuilder'
+import SprungbrettJobModel from '../models/SprungbrettJobModel'
+
+export default new EndpointBuilder('sprungbrett')
+  // todo take the whole url
+  .withStateToUrlMapper(state => `https://webnext.integreat-app.de/proxy/sprungbrett/app-search-internships?location=${state.sprungbrettUrl.url}`)
```

Comment:
Does the sprungbrettUrl need to be in the global state? I think the location should be enough, right?